packages/contracts/test/diamond/DiamondTest.t.sol

```diff
@@ -12,7 +12,7 @@ contract TestDiamond is DiamondSetup {
     }
 
     function testHasMultipleFacets() public {
-        assertEq(facetAddressList.length, 15);
+        assertEq(facetAddressList.length, 16);
     }
```

Comment:
I feel like this test will be tricky if we are merging in new branches/features constantly. Perhaps it should just log the amount of facets? @zgorizzo69